aep/general/0003/aep.md.j2

```diff
@@ -80,6 +80,12 @@ addresses.
 Either a programmatic client or a user that consumes an API. This term should
 be used when a statement refers broadly to both programs and users.
 
+### Collection
+
+A collection represents a discrete set of resources of a specific type. A resource
+may have one or more collections, with each collection having its own set of
+parent collections.
```

Comment:
What does it mean for a collection to have a set of parent collections?  Doesn't each collection have a single parent?

Reply:
if you're a nested resource of depth two, you'll have two collections that you're a child of (e.g. /publishers/books/editions/ would have a parent collection of publisher and a books).

But I agree the wording is confusing, so I'll just say a single parent collection.

Reply:
Oh yeah, I think I'd call non-direct parent collections "ancestor collections".  My great-grandparents aren't my parents, after all...anyway, LGTM.